lib/content/informasi-donasi.ts

```diff
@@ -0,0 +1,15 @@
+import informasiDonasi from "~/_content/donasi/informasi-donasi.json";
```

Comment:
```suggestion
import donasi from "~/_content/donasi.json";
```

Please simplify the file structure, thanks!